site/dat/repo/various.json

```diff
@@ -651,6 +656,13 @@
     "vendor": "adrianmo",
     "markerClass": "io.github.adrianmo.jmeter.backendlistener.azure.AzureBackendClient",
     "versions": {
+      "0.2.1": {
```

Comment:
Same here I guess

Reply:
Same goes here. Or is my PR wrong in some way?

Reply:
Yes I think so. 
Diff should only concern your parts of code not  parts that are not yours.
I think you sync your fork and make a new PR.